### PR TITLE
Python: Highlight py/use-of-input is for Python 2

### PR DESCRIPTION
**Pull request by RasmusWL**
I also want to put into question the part of the documentation that says

> (...) `ast.literal_eval()` can always be used safely

When https://docs.python.org/2/library/ast.html#ast.literal_eval says:

> **Warning:** It is possible to crash the Python interpreter with a sufficiently large/complex string due to stack depth limitations in Python’s AST compiler.

### CODE REVIEW

File: python/ql/src/Expressions/UseofInput.qhelp [hunk 1]
```diff
@@ -3,20 +3,20 @@
   "qhelp.dtd">
 <qhelp>
 <overview>
-<p>A call to the input() function, <code>input(prompt)</code> is equivalent to <code>eval(raw_input(prompt))</code>. Evaluating user input without any checking can be a serious security flaw.</p>
+<p>In Python 2, a call to the input() function, <code>input(prompt)</code> is equivalent to <code>eval(raw_input(prompt))</code>. Evaluating user input without any checking can be a serious security flaw.</p>
```
Comment (tausbn) [inline python/ql/src/Expressions/UseofInput.qhelp]: I feel like `input()` should be enclosed in `code` tags, for consistency.

**Comment by tausbn** [comment]
Ready to merge as far as I'm concerned. 
Any comments from the @Semmle/doc team?

**Comment by RasmusWL** [comment]
> It doesn't look as if you've addressed the question you posed in the PR description, but perhaps you've discussed that offline?

it's a very good point -- I'll talk with Taus about this today!

**Comment by tausbn** [comment]
Thanks for bringing this up, Felicity, we had indeed not discussed it.

I think it's fine as written in this context. Here, the failure mode we're trying to prevent is arbitrary code execution, and in this situation a crash is less critical. (Also, really people should be using Python 3 already!)

**Pull request merged**